tests/stub/routing.py

```diff
@@ -129,8 +129,8 @@ def router_with_bookmarks_script_create_adb(self):
         C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS#}
         S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
         C: BEGIN {"db": "system"}
-        C: RUN "CREATE database foo" {} {}
         S: SUCCESS {}
+        C: RUN "CREATE database foo" {} {}
```

Comment:
I think I've seen many of those, where the stub server expects the driver to pipeline multiple requests. We should probably (in another PR) change that.